cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,20 @@
+import time
+from django.core.management.base import BaseCommand
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until db is available"""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
```

Comment:
if the database does not exist, you create a infinite loop(

Reply:
This solution was on the link on this task)

Reply:
anyway, better doesn't create an infinite loop

Reply:
Ok)
I will try